yksgi/src/lib.rs

```diff
@@ -13,7 +13,8 @@ use std::slice;
 mod llvmbridge;
 use llvmbridge::{get_aot_original, BasicBlock, Module, Type, Value};
 
-static ALWAYS_SKIP_FUNCS: [&str; 6] = [
+static ALWAYS_SKIP_FUNCS: [&str; 7] = [
```

Comment:
I wonder if this can be `[&str; _]` or do we have to hard-code the constant?

Reply:
I wish we could, but alas:

```
16 | static ALWAYS_SKIP_FUNCS: [&str; _] = [
   |                                  ^ `_` not allowed here
```

Also tried omitting it entirely, but that's also invalid:

```
16 | static ALWAYS_SKIP_FUNCS: [&str] = [
   |                           ^^^^^^ doesn't have a size known at compile-time
   |
```

It irritates me too.